src/components/OneArticle.jsx

```diff
@@ -39,6 +94,22 @@ function OneArticle() {
       <br />
       <p>{singleArticle.body}</p>
       <br />
+      <h3>Votes: {singleArticle.votes}</h3>
+      <button
+        className="upvoteButton"
+        onClick={() => handleClick(1, article_id)}
+        disabled={upvoteClick}
+      >
+        Upvote
+      </button>
+      <button
+        className="downvoteButton"
+        onClick={() => handleClick(-1, article_id)}
+        disabled={downvoteClick}
+      >

```

Comment:
Great that you're using the `disabled` property to limit the user's ability to mash the up or down vote button ✅